src/Xrefcheck/Scanners/Symlink.hs

```diff
@@ -0,0 +1,33 @@
+{- SPDX-FileCopyrightText: 2023 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+-- | Symlink files symlinkScanner.
+--
```

Comment:
Autocomplete is living its own life? :sweat_smile:

Also, we generally don't use the blank line between module description and `module ...`.

Reply:
I just tried to follow the same module documentation as in `Scanner.Markdown`. Check it, I think that an AI couldn't  have done it better (nor worse) 🤣

Reply:
Our brain-integrated refactoring tool shows its power :joy:  

Looking at the blame of `Scanner.Markdown` module... You know, with this project I managed to get a completely ridiculous "Who wrote this WTF code?... Ah, so it was me" number of situations, never expected this joke would apply to me so well :rofl: